tests/e2e/e2e_util_test.go

```diff
@@ -16,65 +14,14 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
-	"github.com/ory/dockertest/v3/docker"
+	"github.com/stretchr/testify/require"
 
```

Comment:
Can you explain the desire to switch to this? I am all for it, just would like to know the motivation to do so

Reply:
Since I'm extracting this change from a bigger refactoring PR #1591, I just copied most of the logic over. In #1591,  I did not have access to the suite and had to import `require` directly instead.

I thought I changed it in this PR but, apparently, I missed it. Thanks for catching that, the import is now removed.